helpers/import.sh

```diff
@@ -155,30 +155,30 @@ terraform import "module.$MODULE_LOCAL_NAME.google_project_service.main[15]" "$P
 terraform import "module.$MODULE_LOCAL_NAME.module.client.google_compute_instance.forseti-client" "$PROJECT_ID/us-central1-c/forseti-client-vm-$RESOURCE_NAME_SUFFIX"
```

Comment:
`us-central1-c` should be `$GCE_ZONE` as per: https://github.com/forseti-security/terraform-google-forseti/pull/265/files